iidm-import-export-api/src/main/java/eu/itesla_project/iidm/datasource/Bzip2FileDataSource.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2016, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.iidm.datasource;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Objects;
+import org.jboss.shrinkwrap.impl.base.io.tar.bzip.BZip2CompressorInputStream;
+import org.jboss.shrinkwrap.impl.base.io.tar.bzip.BZip2CompressorOutputStream;
```

Comment:
I considered the "commons-compress".  However, the shrinkwrap impl was already in this  modules' pom dependencies  list (used by other classes)  ......